<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KalignMsaBuilderOptionsWidget</class>
 <widget class="IMsaBuilderOptionsWidget" name="KalignMsaBuilderOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="toolTip">
      <string>Also known as &quot;Sort Order&quot;</string>
     </property>
     <property name="title">
      <string>Output Order:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="orderByInputSequencesRadioButton_">
        <property name="text">
         <string>Input sequences</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="orderByTreeRadioButton_">
        <property name="text">
         <string>Tree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="orderByGapsRadioButton_">
        <property name="text">
         <string>Gaps</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Guide Tree:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="guideByUPGMARadioButton_">
        <property name="text">
         <string>UPGMA</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="guideByNJRadioButton_">
        <property name="text">
         <string>Neighbor joining</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Distance Method:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="distanceWuRadioButton_">
        <property name="text">
         <string>Wu</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="distancePairRadioButton_">
        <property name="text">
         <string>Pair</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gap Penalties:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gap open penalty:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="AgDoubleSpinBox" name="gapOpenPenaltyDoubleSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>11.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Gap extension penalty:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="AgDoubleSpinBox" name="gapExtensionPenaltyDoubleSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Terminal gap penalties:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="AgDoubleSpinBox" name="terminalGapPenaltyDoubleSpinBox_">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.400000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Use default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IMsaBuilderOptionsWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/IMsaBuilderOptionsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AgDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui/widgets/AgDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapOpenPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapExtensionPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>terminalGapPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
